### PR TITLE
Adjust API

### PR DESCRIPTION
**Pull request by nbassler**
Related to #126

### CODE REVIEW

File: pytrip/cube.py [hunk 1]
```diff
@@ -241,7 +249,7 @@ def create_cube_from_equation(self, equation, center, limits, radial=True):
         y = np.linspace(self.dimx - 0.5, 0.5, self.dimx) * self.pixel_size - center[1]
         xv, yv = np.meshgrid(x, y)
 
-    def load_from_structure(self, voi, preset=0, data_type=np.int16):
+    def set_all_voxels_by_voi(self, voi, preset=0, data_type=np.int16):
```
Comment (grzanka) [inline pytrip/cube.py]: I'd name it `mask_by_voi` as it is clear that we deal with voxels here

File: tests/test_let.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def setUp(self):
 
     def test_read(self):
         l = LETCube()
-        l.read_trip_data_file(self.let001)
+        l._read_trip_data_file(self.let001)
```
Comment (grzanka) [inline tests/test_let.py]: This is a private method and shouldn't be used here.

**Comment by nbassler** [comment]
@grzanka better?

**Comment by grzanka** [comment]
@nbassler much better

**Pull request merged**